myoolala/react-starter
# Feat/add terrafrom init script

## Acceptance Criteria
A bash script now exists to do the initial configuration for terraform

## Change Description
There is now a helper script that does the initial setup for terraform to aid new developers

## Verification Instructions
* start the devops container
* cd into the scripts directory
* run the initialization script
* confirm that the terragrunt directory now exists with a secrets file in the fargate folder

## Commit Message
feat: Adds base initialization script to setup terraform and a sops secrets file
